mono/utils/Makefile.am

```diff
@@ -236,7 +236,8 @@ monoutils_sources = \
 	ward.h \
 	options.h \
 	options-def.h \
-	options.c
+	options.c \
+	ftnptr.h
```

Comment:
this file will be deleted by #47758 so probably best to not touch it

Reply:
Its still present in this repo.

Reply:
ah, didn't see that this is mono/mono .... :smile: